### PR TITLE
[irods/irods#7592] add ubuntu 24.04 projects

### PR DESCRIPTION
**Pull request by SwooshyCueb**
In service of irods/irods#7592

Companion PRs:
  - irods/irods#7733
  - irods/irods#7739
  - irods/irods_client_icommands#477
  - irods/irods_client_icommands#478
  - irods/externals#246
  - irods/irods_development_environment#137

### CODE REVIEW

**Comment by SwooshyCueb** [comment]
Ready for review

File: projects/ubuntu-24.04/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        python3-requests \
+        python3-setuptools \
+        rsyslog \
+        systemd \
+        unixodbc \
+    && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/*
+
+# Installing packages via pip in Ubutnu 24.04 is more strict than before. Given
```
Comment (alanking) [inline projects/ubuntu-24.04/Dockerfile:45]: Ubutnu -> Ubuntu

**Review by alanking — COMMENTED** [review]
Looks good! Saw one typo in a comment but otherwise we can go ahead and put this in

File: projects/ubuntu-24.04/Dockerfile (hunk 1 above)
Comment (SwooshyCueb) [reply projects/ubuntu-24.04/Dockerfile]: Fixed

**Review by alanking — APPROVED** [review]
Yep, # it.

**Comment by SwooshyCueb** [comment]
\#'d

**Pull request merged**